packages/template-ui/src/components/Breadcrumb.vue

```diff
@@ -20,19 +20,21 @@
         …
       </li>
       <!-- One li with link per parent item -->
-      <li
-        v-for="p in parentItems"
-        :key="p.id"
-        v-b-tooltip.hover
-        :title="p.title"
-        class="breadcrumb-item text-light text-truncate"
-      >
-        <b-link
-          :to="getTopicUrl(p)"
+      <template v-if="showBreadcrumb">
```

Comment:
What about the "..." above? it seems you also don't want those when `showBreadcrumb` is false.